packages/protocol/contracts/thirdparty/risczero/StructHash.sol

```diff
@@ -0,0 +1,89 @@
+// Copyright 2024 RISC Zero, Inc.
```

Comment:
Same as above. Not needed to bring it over as a copy, reference with `npm` or `git` `submodule`.

Reply:
If we can specify a release version in the dependency URL, it should be fine but please don't use its default branch for dependency.